src/panoramic/cli/field_mapper.py

```diff
@@ -54,3 +55,22 @@ def map_field_from_local(field: PanoField) -> Field:
         calculation=field.calculation,
         display_format=field.display_format,
     )
+
+
+def map_column_to_field(column: Dict[str, str], is_identifier: bool = False) -> PanoField:
+    aggregation = {'type': column['aggregation_type']} if column.get('aggregation_type') is not None else None
+    field_type = 'dimension' if is_identifier else column['taxon_type']
+
+    assert len(column['field_map']) == 1
+    slug = column['field_map'][0]
+
+    return PanoField.from_dict(
```

Comment:
Why the `from_dict(dict(...))` instead of just passing the kwargs into `PanoField` ?

Reply:
You don't have to specify the non-defaulted attrs this wat - but come to think of it, maybe it'd be better to add `None` deafaults to the `Optional` attrs and use a constructor directly.

Reply:
Definitely agree with that

Reply:
Fixed in 92dbcfceb8c77d413918abc9de4cdd9cf231e581.